util/github_util.go

```diff
@@ -0,0 +1,21 @@
+package util
+
+import (
+	"fmt"
+
+	"github.com/go-resty/resty/v2"
+
+	"github.com/thedevsaddam/gojsonq/v2"
+)
+
+func GetLatestTag(githubPAT, githubOrg, repo string) (string, error) {
```

Comment:
Was this source file committed by accident? It is unused and looks very similar to the one added in #18.

Reply:
I will address this..

Reply:
This file was not accidentally uploaded; we are making use of this function in ```cmd/util.go```